modules/ROOT/pages/_partials/endpoint-as2-receive.adoc

```diff
@@ -13,7 +13,7 @@ Required: Yes +
 Required: No +
 
 | AS2 URL (Automatically generated at time of deployment)
-| The HTTP(S) address at which you will be receiving AS2 messages.
+| The HTTP(S) address at where you receive AS2 messages.
```

Comment:
```suggestion
| The HTTP or HTTPS address where you receive AS2 messages.
```
Also, do not end such a fragment with a period unless this is part of a list in which one or more descriptions is a complete sentence.

Reply:
Thanks.